app/models/application.rb

```diff
@@ -1,4 +1,13 @@
 class Application < ApplicationRecord
   has_many :application_pets 
   has_many :pets, through: :application_pets
+
+  validates :name, presence: true
```

Comment:
So this is preventing an uncomplete form from being saved?

Reply:
that is exactly what it is doing!